scipy/stats/_multivariate.py

```diff
@@ -806,18 +810,52 @@ def fit(self, x):
             Maximum likelihood estimate of the covariance matrix
 
         """
-        # input validation
+        # input validation for data to be fitted
         x = np.asarray(x)
         if x.ndim != 2:
             raise ValueError("`x` must be two-dimensional.")
 
         n_vectors, dim = x.shape
 
+        # convenience variables for checking if parameters are fixed
+        mean_fixed = fix_mean is not None
+        cov_fixed = fix_cov is not None
+
+        # if both parameters are fixed, there is nothing to fit
+        if mean_fixed and cov_fixed:
+            error_msg = "Both parameters are fixed. There is nothing to fit."
+            raise ValueError(error_msg)
```

Comment:
Let's just return the fixed values. I think it is safe to depart from the unfortunate precedent of the existing univariate interface here because 1) multivariate distributions are different and 2) the new univariate interface will.

Advantage: self-consistency / no special case to test.

However, I do think this should happen after input validation.